### PR TITLE
Fix coverage tests ordering and small improvements

### PR DESCRIPTION
**Pull request by ihalaij1**
**What?**
Tests generated in `testcoveragemeta.py` are now properly ordered in the html feedback when more than 9 tests exist by changing the names of the generated tests from format `'test_coverage_{}'.format(test_num)` to `'test_coverage_{:02d}'.format(test_num)`.

Various other small changes to `testcoveragemeta.py` improve the code style and readability (imports in alphabetical order, better grouping of code, newlines). An unused import (`imp`) was also removed.

Default test result messages `The test was a success!`, `The test failed, reason:`, `An error occurred:` are moved from `testcoveragemeta.py` to `graderunittest.py`.

These messages are now shown by default in the html feedback on all tests instead of only in coverage tests. There is no reason for grader-utils not to provide these default messages (they can still be overridden in the grader unittests, if the test author so wishes). 

This also makes expanding tests that succeeded less weird by actually saying something, instead of just showing an empty box:
image

Changed two `Exception`s in `graderunittest.py` to class `GraderUtilsError`. This error fits better and is more consistent with other parts of grader-utils, where `GraderUtilsError` is already used.

Previously, in `default.css`, default stylesheet badge colors in cases of errors had to be overridden to be gray, or else they would be black. Recent updates to the A+ stylesheet have forced all (error, success, fail) badge colors to be gray. This means that we have to override all the colors for them to be correct (gray, green, red). If the overrides are removed all points badges turn to black.

This could probably be fixed in the stylesheet. I don't think black badges are used anywhere in A+.
`_panels.scss` seems to override `.panel-default > .panel-heading .badge` to black for some reason.

### CODE REVIEW

File: graderutils_format/templates/default.css [hunk 1]
```diff
@@ -25,8 +25,21 @@ pre {
...
+/* Override stylesheet badge colors inside panel-heading since otherwise they are black */
+
+/* Error */
 div.grading-task .panel-default > .panel-heading .badge {
     color: #fff;
     background-color: #777;
 }
+/* Success */
+div.grading-task .panel-default > .panel-heading .badge.badge-success {
+    color: #fff;
```
Comment (Mankro) [inline graderutils_format/templates/default.css:38]: I verified with the WAVE plugin that these colours pass the colour contrast requirement for accessibility.

**Review by Mankro — APPROVED** [review]
Excellent! I rebased this myself onto the upstream master and moved that one `import coverage` line.

**Comment by Mankro** [comment]
About the CSS: `_panels.scss` comes from Bootstrap so we are not going to modify that file directly. If there is a suitable Sass variable that could be overridden to change the colour, then the variable can be set in the A+ code. The unwanted styles override the badge colours set in A+ because these rules are more specific (targeting only panel headings instead of any badge class). A+ badges don't have the `!important` modifier since its use is discouraged.

https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity

This is not very important, though.
